File: LeetCode/Easy/1672 - Richest Customer Wealth/1672 - Richest Customer Wealth.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
 var maximumWealth = function(accounts) {
    let ret = -Infinity;
    for (const account of accounts) {
        let sum = 0;
        for (const i of account) sum += i;
        ret = Math.max(ret, sum);
    }
    return ret;
};

File: LeetCode/Easy/1672 - Richest Customer Wealth/1672 - Richest Customer Wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int maxWealth = 0;
        for (auto account : accounts) {
            int cur = 0;
            for (auto acc : account) {
                cur += acc;
            }
            maxWealth = max(maxWealth, cur);
        }
        return maxWealth;
    }
};